minecraft/src/main/java/net/fabricmc/loader/impl/game/minecraft/McVersionLookup.java

```diff
@@ -287,7 +287,9 @@ protected static String getRelease(String version) {
 			int year = Integer.parseInt(matcher.group(1));
 			int week = Integer.parseInt(matcher.group(2));
 
-			if (year >= 24 && week >= 33) {
+			if (year >= 24 && week >= 44) {
```

Comment:
Actually, this will break with a 25w01a. (it was wrong before so not your fault)

```suggestion
			if (year == 24 && week >= 44 || year >= 25) {
```
I think this will be correct, and save the need for a release next year if this snapshot series goes into the new year.

Reply:
I vaguely remember that they aimed for a holiday release (I'd assume this year)?

Reply:
I would think so too, but lets not count on it.